pydis_site/apps/content/resources/guides/pydis-guides/help-channel-guide.md

```diff
@@ -10,66 +10,84 @@ relevant_links:
 
 On the 5th of April 2020, we introduced a new help channel system at Python Discord. This article is a supplementary guide to explain precisely where to go to find help.
 
-We have two different kinds of help channels in our community - **Topical help channels**, and **general help channels**.
-Where you should go depends on what you need help with.
-These channels also attract different helpers, and move at different speeds, which affects the kind of help you're likely to receive, and how fast you get that help.
+We have two different kinds of help channels in our community: **topical help channels** and **general help channels**.
+The topical channels can be great for longer and broader discussions, whereas a general help channel might be more appropriate for specific troubleshooting. Individual topical channels may have a narrower audience than the general help channels, but that may also be to your advantage if the audience is very familiar with the area of Python you are struggling with.
```

Comment:
I'm not sure I like this description, it's a bit confusing. If I were to describe it, I'd just say that the help system provides channels for focused help to you individually, and can have any python related question, while the topical channels have specific topics, aren't reserved for one person at a time, but might have a better chance to been seen by someone with domain knowledge.

Reply:
I agree, reading it now it is confusing hmmmmmmmmmm.

Reply:
How's this @mbaruh ?

In topical help channels, many users can discuss and ask for help around specific domains or areas of Python; anyone can participate in giving and asking help. General help channels can be used for all Python-related help, and are temporarily claimed and reserved by individual users so that they can get focused attention to their problem.

Reply:
Removing this introduction section and adding the information to their respective sections as needed. Adding a table of contents instead.